files/image_config/logrotate/rsyslog.j2

```diff
@@ -85,7 +85,8 @@
             if [ $USED_KB -lt $THRESHOLD_KB ]; then
                 break
             else
-                OLDEST_ARCHIVE_FILE=$(find /var/log -type f -printf '%T+ %p\n' | grep -E '.+\.[0-9]+(\.gz)?$' | sort | head -n 1 | awk '{ print $2; }')
+                SORTED_ARCHIVE_FILES=$(find /var/log -type f -printf '%T+ %p\n' | grep -E '.+\.[0-9]+(\.gz)?$' | sort )
```

Comment:
@assrinivasan actually since we want only the 1st sorted we can use awk itself like so
` strace -f -e execve,write bash -c 'ls -l /usr/bin/ /var/lib /usr/lib | sort | awk 'NR == 1 {print $2}''`

There is no need to use `head -n 1`

Reply:
Done in latest commit.